src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -39,8 +43,38 @@
      * Андрей - 600
      * София - 900</p>
      */
+
+    public static final DateTimeFormatter DATEPATTERN = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
if constant name contains a few words then we are separating them with underscore